test/routes/events.test.js

```diff
@@ -1,97 +1,145 @@
 const { assert } = require('chai');
 const { HttpError, framework, router } = require('@ewarren/serverless-routing');
-const { mockAwsPromise, s3ify } = require('../stubs');
+const {
+  connectToDatabase,
+  initDatabase,
+  closeDatabaseConnection
+} = require('../../src/utils/connectToDatabase');
+const DatabaseCleaner = require('database-cleaner');
 const eventsRoutes = require('../../src/routes/events');
 
-describe('test upcoming events route', function() {
-  it('should return the latest events in order', function(callback) {
-    const testStub = {
-      getObject: mockAwsPromise({
-        Body: s3ify({
-          data: [
-            {
-              'Event Title (US-EN)': '1',
-              'Published': true,
-              'Date': Date.now() + (1000 * 60 * 60 * 24),
-            },
-            {
-              'Event Title (US-EN)': '2',
-              'Published': true,
-              'Date': Date.now() + (1000 * 60 * 60 * 24 * 2),
-            },
-          ],
-        }),
-      }),
-    };
+describe('events routes', function() {
+  let onRequest = null;
+  let testDb = null;
+
+  const testEvents = [{
```

Comment:
Love it! Awesome

Reply:
fyi I moved the test event data into a fixture file!